### PR TITLE
Android - Segmentation Mask DIS

### PR DESCRIPTION
**Pull request by st-nhatnguyen**

### CODE REVIEW

File: examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+            bitmap.height
+        )
+    }
+
+    enum class Delegate {
+        CPU, NNAPI
+    }
+
+    enum class Model(val title: String, val fileName: String) {
+        ISNET_TFL_DRQ("isnet_1", "isnet_tfl_drq.tflite"),
```
Comment (PaulTR) [inline examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt:176]: This only needs isnet_tfl_drq.tflite, you can remove the selector for the other two models.

File: examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        try {
+            val percent = bitmap.width.toFloat() / bitmap.height.toFloat()
+            withContext(Dispatchers.IO) {
+                if (interpreter == null) return@withContext
+                val startTime = SystemClock.uptimeMillis()
+
+                val (_, h, w, _) = interpreter?.getInputTensor(0)?.shape() ?: return@withContext
+                val imageProcessor =
+                    ImageProcessor.Builder()
+                        .add(ResizeOp(h, w, ResizeOp.ResizeMethod.BILINEAR))
```
Comment (PaulTR) [inline examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt:93]: When resizing the image to fit on the screen, is there a way to sizes it so it isn't cropping?

Comment (PaulTR) [reply examples/image_segmentation_DIS/android/app/src/main/java/com/example/segmentationdis/ImageSegmentationHelper.kt:93]: For example, I was playing with this using a different background to test different fabric colors for cross-stitching patterns. The image taken by the camera is cropped, zoomed, and off center when displayed, however.  https://photos.app.goo.gl/T3Yi3j131NxP4d4k8

**Comment by st-nhatnguyen** [comment]
Hi @PaulTR .

I have solved your comments, please help me review again. 

Thank you 🙇

**Pull request merged**